### PR TITLE
User authentication

### PR DESCRIPTION
**Pull request by zanehirning**
User authentication working, groups created. Initial signup makes the user a customer. Login/signup pages are styled and the flow is complete

### CODE REVIEW

File: DroneCones/DroneConesApp/forms/forms.py
Comment (bnsmth235) [inline DroneCones/DroneConesApp/forms/forms.py:1]: Do we have the two different forms.py files on purpose?

File: DroneCones/DroneConesApp/templates/DroneConesApp/landing.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{% block content %}
+<body>
+<div class="full-container">
+    <h2>DroneCones: Our Story</h2>
+<div class="container"><h2>Checkout Our New Flavor: Mint Chocolate Chip</h2><button>Order Now!</button></div>
+<div class="container"><p>About</p></div>
+</div>
+</body>
+{% endblock %}
+>>>>>>> e3d60e8d9cdcf7059a9b82ceb667f5ed290d5e0c
```
Comment (bnsmth235) [inline DroneCones/DroneConesApp/templates/DroneConesApp/landing.html:19]: Looks like this is left over from a merge or something

**Review by bnsmth235 — CHANGES_REQUESTED** [review]

File: DroneCones/DroneConesApp/forms/forms.py
Comment (zanehirning) [reply DroneCones/DroneConesApp/forms/forms.py:1]: I did not catch that. I think that happened when we were messing around on the user-auth branch deleting and changing the base.html. I can clean that up before we merge.

**Review by bnsmth235 — APPROVED** [review]

File: DroneCones/DroneConesApp/templates/DroneConesApp/landing.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{% extends 'DroneConesApp/misc/base.html' %}
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="{% static 'DroneConesApp/mainStyle.css' %}">
+    <script src="{% static 'DroneConesApp/final.js' %}" defer></script>
+ <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+{% block content %}
```
Comment (hamnerben) [inline DroneCones/DroneConesApp/templates/DroneConesApp/landing.html:10]: Do you think you could explain to me the `block content` usage?  I thought extending `base.html` would put the entire file in the body of `base.html` therefore no head is needed in any file that extends `base.html`?

**Review by hamnerben — COMMENTED** [review]
I just had one question on the template I put in the code comments

File: DroneCones/DroneConesApp/templates/DroneConesApp/landing.html (hunk 2 above)
Comment (bnsmth235) [reply DroneCones/DroneConesApp/templates/DroneConesApp/landing.html:10]: ^^

File: DroneCones/DroneConesApp/static/DroneConesApp/mainStyle.css [hunk 3]
```diff
@@ -66,12 +63,21 @@ pre {
     padding: 10px;
     gap: 10px;
     display: flex;
+<<<<<<< HEAD
```
Comment (hamnerben) [inline DroneCones/DroneConesApp/static/DroneConesApp/mainStyle.css:66]: Here is another merge conflict comment in the code

File: DroneCones/DroneConesApp/templates/DroneConesApp/landing.html (hunk 2 above)
Comment (zanehirning) [reply DroneCones/DroneConesApp/templates/DroneConesApp/landing.html:10]: I can fix this. I didn't necessarily do anything in the landing page, but Porter did some linking stuff to the sign in pages on this branch. I think that this is just a result.

**Comment by zanehirning** [comment]
Yeah, I can fix that. With all the changes to main I didn't get to covering all of my bases it appears. I can fix that tonight

**Review by hamnerben — APPROVED** [review]

**Pull request merged**